awx/main/models/rbac.py

```diff
@@ -733,5 +735,83 @@ def sync_parents_to_new_rbac(instance, action, model, pk_set, reverse, **kwargs)
             give_or_remove_permission(child_role, team, giving=is_giving)
 
 
+ROLE_DEFINITION_TO_ROLE_FIELD = {
+    'Organization Member': 'member_role',
+    'Controller Organization Member': 'member_role',
+    'WorkflowJobTemplate Admin': 'admin_role',
+    'Organization WorkflowJobTemplate Admin': 'workflow_admin_role',
+    'WorkflowJobTemplate Execute': 'execute_role',
+    'WorkflowJobTemplate Approve': 'approval_role',
+    'InstanceGroup Admin': 'admin_role',
+    'InstanceGroup Use': 'use_role',
+    #  'ExecutionEnvironment Admin': '',
```

Comment:
no roles on EE - not sure how to handle this one

Reply:
These are new, assignments to those should not change anything in the old system.